### DISCUSSION TITLE
Allow changing repositories star color

### DISCUSSION THREAD

**Discussion by markkrj** [community/community#8659] [Other Feature Feedback, Questions, & Ideas]
It would allow to immediately know in what list a starred repo is.

Edit: Well, we can put repositories in more than one list, so, maybe, we could get an option to change the star color by repository, or just change it altogether, as I preferred the light-grey for the dark theme (yellow seems odd in dark theme).

**Comment by brianlovin** [comment]
Thanks for the suggestion @markkrj — I agree that right now it's not super obvious to know what lists a repo is in, even after you've starred it. Having to open the dropdown to check is a bit of a pain. I'll pass this on to the team to think about solutions 😊
